mne/preprocessing/_css.py

```diff
@@ -0,0 +1,109 @@
+# Author: John Samuelsson <johnsam@mit.edu>
+
+import numpy as np
+import mne
+from mne.utils import logger
+from numpy import linalg
+
+
+def temp_proj(ref_2, ref_1, raw_data, n_proj=6):
```

Comment:
do not make this function public

temp_proj -> _temp_proj

and remove it from API reference list

Reply:
The temp_proj function is meant to be public because it can be used for many other purposes than css

Reply:
I think one can justify the function being public if it's used in the example. Otherwise, safer to leave it private for now